go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/gorilla/mux v1.8.1
```

Comment:
Cannot comment on that line but ibc-go can be updated.
Other bumps can also be a part of this PR or a PR that merges to this branch.